docs/more-info/build-server-setup/tfs-build-vnext.md

```diff
@@ -0,0 +1,26 @@
+# Visual Studio Online (Build vNext) Setup
+## Basic Usage
+In [Visual Studio Online](https://www.visualstudio.com/) build vNext (the web based build system) you can add a build step as follows:
+
+* **Build Step:** Command Line
+	* Might need to fully qualify the path.
+	* Tip: Use a script and/or another command step to call `NuGet Install GitVersion.CommandLine` first so you don't have to check in the exe.
+* **Tool:**  `GitVersion.exe`
+* **Arguments:** `/output buildserver /updateassemblyinfo true`
+
+Then in your build parameters simply [add a placeholder](#nuget-in-teamcity) of the GitVersion variables you would like to use.
```

Comment:
This seems like copy/past from TeamCity documentation and not accurate for this PR